Foundation/NSXMLParser.swift

```diff
@@ -233,9 +233,9 @@ internal func _NSXMLParserStartElementNs(ctx: _CFXMLInterface, localname: Unsafe
     let parser = ctx.parser
     let reportQNameURI = parser.shouldProcessNamespaces
     let reportNamespaces = parser.shouldReportNamespacePrefixes
-    let prefixLen = prefix == nil ? strlen(UnsafePointer<Int8>(prefix)) : 0
+    let prefixLen = prefix == nil ? UInt(strlen(UnsafePointer<Int8>(prefix))) : 0
```

Comment:
isn't the declaration of strlen
`size_t strlen(const char *s)`
which size_t should be unsigned in any reasonable environment typedefed by `__SIZE_TYPE__` hopefully or `unsigned long` if that isn't the case

Which os defines it as `int strlen(const char *s)`?